app/controllers/api/subcollections/snapshots.rb

```diff
@@ -25,8 +23,7 @@ def snapshots_create_resource(parent, _type, _id, data)
       end
 
       def delete_resource_snapshots(parent, type, id, _data)
-        validation = parent.validate_remove_snapshot(id)
```

Comment:
@durandom Yes, there are some changes in manageiq-ui-classic, I have created separate PR for that, it's still in WIP because I see that failing till this PR is not merged. How are we maintaining related changes between two repos? Is there any way to link these related PRs?